src/types/TransactionReceipt.ts

```diff
@@ -0,0 +1,23 @@
+import {TransactionReceipt as ProtoTransactionReceipt} from "../generated/TransactionReceipt_pb";
+import {AccountId, accountIdToSdk} from "./AccountId";
+import {ContractId, contractIdToSdk} from "./ContractId";
+import {FileId, fileIdToSdk} from "./FileId";
+import {ExchangeRateSet, exchangeRateSetToSdk} from "./ExchangeRate";
+
+export type TransactionReceipt = {
+    status: number;
+    accountId: AccountId | null;
+    fileId: FileId | null;
+    contractId: ContractId | null;
+    exchangeRateSet: ExchangeRateSet | null;
+}
+
+export function receiptToSdk(receipt: ProtoTransactionReceipt): TransactionReceipt {
+    return  {
+        status: receipt.getStatus(),
+        accountId: receipt.getAccountid() == null ? null : accountIdToSdk(receipt.getAccountid()!),
```

Comment:
This could be simplified to `receipt.getAccountid() && accountIdToSdk(receipt.getAccountid()!)`

Is there a reason you're not using that form?